File: Selenium/Activities/Activity22.java
import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity22 {
	public static void main(String args[]) {
		WebDriver driver;
		driver = new FirefoxDriver();
		WebDriverWait wait=new WebDriverWait(driver,Duration.ofSeconds(10));

		driver.get("https://training-support.net/webelements/popups");
		System.out.println("the title of the page is: "+driver.getTitle());
		WebElement popUp = driver.findElement(By.cssSelector("#launcher"));
		popUp.click();
		wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("#username")));
		WebElement userName = driver.findElement(By.cssSelector("#username"));
		userName.sendKeys("admin");
		WebElement password = driver.findElement(By.cssSelector("#password"));
		password.sendKeys("password");
		
		WebElement login = driver.findElement(By.cssSelector("button.svelte-1pdjkmx"));
		login.click();
		
		WebElement message = driver.findElement(By.cssSelector(".mt-5"));
		System.out.println("The confirmation message is: "+message.getText());
		
		driver.quit();
	}

}

File: Selenium/Activities/Activity22.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.select import Select
with webdriver.Firefox() as driver:
    wait=WebDriverWait(driver,timeout=10)
    driver.get("https://training-support.net/webelements/popups")
    print("the title of the page is: " ,driver.title)
    popUp = driver.find_element(By.CSS_SELECTOR,"#launcher")
    popUp.click()

    wait.until(EC.visibility_of_element_located((By.CSS_SELECTOR, "#username")))
    userName = driver.find_element(By.CSS_SELECTOR,"#username")
    userName.send_keys("admin")
    password = driver.find_element(By.CSS_SELECTOR,"#password")
    password.send_keys("password")
    login = driver.find_element(By.CSS_SELECTOR,"button.svelte-1pdjkmx")
    login.click()
    message = driver.find_element(By.CSS_SELECTOR,".mt-5")
    print("The confirmation message is: ",message.text)
    driver.quit()
